src/infrastructure/figma/figma-service.ts

```diff
@@ -51,15 +49,15 @@ export class FigmaService {
 	};
 
 	/**
-	 * Return Atlassian design for the Figma design with the given ID.
+	 * Returns Atlassian design for the Figma design with the given ID if it exists; otherwise -- `null`.
 	 *
 	 * @throws {DesignNodeNotFoundError} Design node referenced by the designId was not found.
```

Comment:
minor: Does this method still throw a DesignNodeNotFoundError? If no, should we delete this comment?

Reply:
Oh, great catch, @elainefigma . Fixed.